index.js

```diff
@@ -17,7 +17,7 @@ var Notify = React.createClass({
 	addNotify: function(title, msg, time, theme) {
 		var key = this.key++;
 		this.state[key] = { title: title, msg: msg, time: time, theme: theme };
-		this.setState(this.state);
+		this.forceUpdate();
```

Comment:
This is wrong, you only want to force update if you make changes and don't want to set the state. But here the state is being changed, so set state is correct. Force update will ignore shouldComponentUpdate.
https://facebook.github.io/react/docs/component-api.html#forceupdate